page_objects/call_interaction_page.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, driver=None):
         self.caller_contact = "//span[@class='info-contact']"
         self.interaction_tab = "//li[contains(@id, 'context')]/a"
         self.history_tab = "//li[contains(@id, 'history')]/a"
-        self.script_tab = "//li[contains(@id, 'script')]/a"
+        self.script_tab = "//li[contains(@id, 'script')]/a | //button[@id='script_btn']"
```

Comment:
you can use the `script_button` from `adapter_page.py`

Reply:
forgot to removed this change. Fixed.